utilities/ldap_sync.sql

```diff
@@ -159,32 +931,40 @@ end
 dcl = query([[
 
 WITH 
-
+---------------------------------------
 get_ad_group_members AS (
--- This CTE will get the list of members in LDAP for each role that contains a comment
-		SELECT  
-		EXA_TOOLBOX.GET_AD_ATTRIBUTE(:l, ROLE_COMMENT, :g)
+---------------------------------------
+/* This CTE will get the list of members in LDAP for each role that contains a comment */
+
+		/*snapshot execution*/ SELECT  
```

Comment:
I don't think /*snapshot execution*/ works here, as it is in the middle of a complex statement, you need to put it at the very beginning (in front of the WITH).